content/blog/2025-08-28-enabling-crb-by-default-for-almalinux10.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Enabling CRB by default for AlmaLinux 10"
+type: blog
+author:
+ name: "Neal Gompa"
+ bio: "Principal Consultant, Velocity Limitless; AlmaLinux Engineering Steering Committee member"
+ image: /users/nealgompa.png
+date: '2025-08-28'
+images:
+  - /blog-images/2025/crb_enabled_almalinux10.png
+post:
+    title: "Enabling CRB by default for AlmaLinux 10"
+    image: /blog-images/2025/crb_enabled_almalinux10.png
+
+---
+
+AlmaLinux OS 10 and AlmaLinux OS Kitten 10 will have the CRB repository enabled by default.
```

Comment:
As both of these were already realeased, we need to include an answer to these questions: 

* Starting when? 
* if I already have 10 installed, what will happen? 
* if I don't want to enable it (or want to disable it after my install), how can I do that?

Reply:
These should be addressed now.